distributed-databases/trino/variables.tf

```diff
@@ -0,0 +1,41 @@
+variable "name" {
+  description = "Name of the VPC and EKS Cluster"
+  default     = "trino-on-eks"
+  type        = string
+}
+
+variable "region" {
+  description = "Region"
+  type        = string
+  default     = "us-west-2"
+}
+
+variable "namespace" {
+  description = "Namespace for Trino"
+  type        = string
+  default     = "trino"
+}
+
+variable "trino_sa" {
+  description = "Service Account name for Trino"
+  type        = string
+  default     = "trino-sa"
+}
```

Comment:
You can use local variables or hard code these within `trino.tf` file and remove from variables

Reply:
addressed